agents/database-agent/setup-database.sh

```
@@ -0,0 +1,51 @@
+    echo "   Products: $(sqlite3 data/example.db "SELECT COUNT(*) FROM products;")"
+    echo "   Orders: $(sqlite3 data/example.db "SELECT COUNT(*) FROM orders;")"
+    
+    echo ""
+    echo "🎉 Database setup complete!"
+    echo ""
+    echo "You can now run the Database Interaction Agent with:"
+    echo "  npm start -- --agent agents/database-agent.yml"
+    echo ""
+    echo "Example interactions you can try:"
+    echo "  - 'Show me all users'"
+    echo "  - 'List products under $100'"
```

Shellcheck (0.10.0)

[error] Braces are required for positionals over 9, e.g. ${10}.

(SC1037)